libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/timeline/item/event/TimelineEventContentMapper.kt

```diff
@@ -34,84 +34,91 @@ import io.element.android.libraries.matrix.impl.media.map
 import io.element.android.libraries.matrix.impl.poll.map
 import org.matrix.rustcomponents.sdk.TimelineItemContent
 import org.matrix.rustcomponents.sdk.TimelineItemContentKind
+import org.matrix.rustcomponents.sdk.use
 import org.matrix.rustcomponents.sdk.EncryptedMessage as RustEncryptedMessage
 import org.matrix.rustcomponents.sdk.MembershipChange as RustMembershipChange
 import org.matrix.rustcomponents.sdk.OtherState as RustOtherState
 
 class TimelineEventContentMapper(private val eventMessageMapper: EventMessageMapper = EventMessageMapper()) {
 
-    fun map(content: TimelineItemContent): EventContent = content.use {
-        when (val kind = it.kind()) {
-            is TimelineItemContentKind.FailedToParseMessageLike -> {
-                FailedToParseMessageLikeContent(
-                    eventType = kind.eventType,
-                    error = kind.error
-                )
+    fun map(content: TimelineItemContent): EventContent {
+        return content.use { _ ->
```

Comment:
It's a bit weird to ignore the parameter with `_` and then user the original value. I mean, it's valid, but a bit confusing.

Reply:
Yeah but I don't like to use `it` and I can't find suitable name not confusing neither...

Reply:
```suggestion
        return content.use {
```

works as well.

Reply:
You're right, less confusing.